src/handler/getPersonList/index.ts

```diff
@@ -1,10 +1,18 @@
 import { APIGatewayProxyResult } from 'aws-lambda';
 
-import { getPersonList } from '../../pseudoDB/index';
+import { getPersonList } from '../../services/person';
 
 export const handler = async (): Promise<APIGatewayProxyResult> => {
+  const res = await getPersonList();
+
+  if (res.data) {
```

Comment:
Seria melhor tratar se a resposta tem o status code esperado antes de continuar.

Reply:
Não sei se entendi bem, mas essa função retorna os dados em 'data' caso não ocorra nenhum erro, se o 'data' não for null o valor é os dados que vão ser retornados.

Reply:
Ah, me desculpa... Eu quis dizer sobre o retorno do método getPersonList(), não dessa função do handler mesmo.